app/controllers/article_controller.rb

```diff
@@ -186,4 +186,8 @@ def setup_eds_session(session)
       debug: Settings.EDS_DEBUG
     ).session_token
   end
+
+  def has_search_parameters?
```

Comment:
@jkeck I'm wasn't really sure where to put this one. In Blacklight, the method lives in a concern [catalog.rb](https://github.com/projectblacklight/blacklight/blob/master/app/controllers/concerns/blacklight/catalog.rb#L112). The app functions when the method is only in the controller. But when I run the view tests, I get the following error:

```
  1) article/index.html.erb home page
     Failure/Error: <% if has_search_parameters? %>

     ActionView::Template::Error:
       super: no superclass method `has_search_parameters?' for #<#<Class:0x007fd2ff512488>:0x007fd3001dfaa8>
       Did you mean?  has_range_limit_parameters?
     # ./app/views/article/index.html.erb:1:in `_app_views_article_index_html_erb___3943846516671069103_70272113930040'
     # ./spec/views/article/index.html.erb_spec.rb:4:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # NoMethodError:
     #   super: no superclass method `has_search_parameters?' for #<#<Class:0x007fd2ff512488>:0x007fd3001dfaa8>
     #   Did you mean?  has_range_limit_parameters?
     #   ./app/views/article/index.html.erb:1:in `_app_views_article_index_html_erb___3943846516671069103_70272113930040'
```